File: internal/service/i18n.go
// SPDX-FileCopyrightText: 2026-present Brian Wang <wangbuke@gmail.com>
// SPDX-License-Identifier: LGPL-3.0-or-later

package service

import (
	i18nservice "github.com/choysum-dev/choysum/internal/i18n/service"
	"google.golang.org/grpc"
)

func (s *ApplicationService) i18nServiceDesc() (*grpc.ServiceDesc, error) {
	return i18nservice.New(s.name, s.runtimeScope).ServiceDesc()
}

File: modules/meta/service/i18n.ts
// SPDX-FileCopyrightText: 2026-present Brian Wang <wangbuke@gmail.com>
// SPDX-License-Identifier: Apache-2.0

import { createTranslate } from '@/core/service/i18n';

/** Meta service terminology binder (module owner = meta). */
const translate = createTranslate('meta');
export const _t = translate._t;
export const _lt = translate._lt;
